src/gui/properties/propertieswidget.cpp

```diff
@@ -173,6 +175,39 @@ PropertiesWidget::~PropertiesWidget()
     delete m_ui;
 }
 
+void PropertiesWidget::displayFileListHeaderMenu()
+{
+    QMenu *menu = new QMenu(this);
+    menu->setAttribute(Qt::WA_DeleteOnClose);
+    menu->setTitle(tr("Column visibility"));
```

Comment:
Will it be displayed on screen? AFAIK my linux environment and Windows doesn't show it and besides qbt menus never uses it.
If not, then it should be removed.

Reply:
Removed.